torch/csrc/jit/fuser/cpu/dynamic_library_win.cpp

```diff
@@ -7,16 +8,29 @@ namespace jit {
 namespace fuser {
 namespace cpu {
 
+
 DynamicLibrary::DynamicLibrary(const char* name) {
   // NOLINTNEXTLINE(hicpp-signed-bitwise)
-  AT_ERROR("NYI: DynamicLibrary on Windows");
+  HMODULE theModule = LoadLibraryA(name);
+  if (theModule)
```

Comment:
Can you add braces for these if statements to match the style of other code?

Reply:
Done